group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupCoordinatorConfig.java

```diff
@@ -76,7 +76,9 @@ public class GroupCoordinatorConfig {
 
     public static final String GROUP_COORDINATOR_REBALANCE_PROTOCOLS_CONFIG = "group.coordinator.rebalance.protocols";
     public static final String GROUP_COORDINATOR_REBALANCE_PROTOCOLS_DOC = "The list of enabled rebalance protocols. Supported protocols: " +
-            Arrays.stream(Group.GroupType.values()).map(Group.GroupType::toString).collect(Collectors.joining(",")) + ". " +
+            Arrays.stream(Group.GroupType.values())
+                    .filter(protocol -> !protocol.equals(Group.GroupType.UNKNOWN))
+                    .map(Group.GroupType::toString).collect(Collectors.joining(", ")) + ". " +
```

Comment:
I wonder whether we should just drop this part from the documentation given that the values are provided automatically. What do you think?

Reply:
Literally speaking, "unknown" is a recognizable type. But physically, "unknown" is not a supported protocol. In the user documentation, we should hide abnormal settings.
Thus I think we should remove it on the document, WDYT?

Reply:
Yeah, we should definitely hide unknown. My point was more about removing explicitly mentioning the types in the doc string as they are automatically displayed. We basically show the same list twice in your screenshot.

Reply:
@m1a2st we can remove the description "Supported protocols ..." from `GROUP_COORDINATOR_REBALANCE_PROTOCOLS_DOC` as the "Valid Values" has displayed the correct values already.